openedx/core/djangoapps/schedules/content_highlights.py

```diff
@@ -33,17 +41,38 @@ def get_week_highlights(user, course_key, week_num):
     """
     if not COURSE_UPDATE_WAFFLE_FLAG.is_enabled(course_key):
         raise CourseUpdateDoesNotExist(
-            "%s does not have Course Updates enabled.",
+            "%s does not have Course Updates waffle flag enabled.",
             course_key
         )
```

Comment:
The above lines in this function can be deleted since they are now in `_get_course_with_highlights`.